{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env sh
 {% include 'misc/header.py' %}
+
+# TODO: Pass the services required by your module to the
+# docker-services-cli e.g. `docker-services-cli up es postgresql redis`
+docker-services-cli up
 pydocstyle {{ cookiecutter.package_name }} tests docs && \
-isort -rc -c -df && \
+isort {{ cookiecutter.package_name }} tests --check-only --diff && \
```

Comment:
We have decided here https://github.com/inveniosoftware/pytest-invenio/issues/56#issuecomment-680726765 that all commands will have the `python -m` prefix so the `run-tests.sh` is more robust regarding reproducibility, more https://github.com/inveniosoftware/invenio-db/pull/131#discussion_r475685849 (one needs to unfold the resolved conversation to see the comment).

Reply:
Ah my bad, thought it was only for the `pytest`. Could there be an RFC/CodiMD for this? Just to have it (spread) clear?

Reply:
@diegodelemos Fixed :) Now is just waiting for pytest to be released. :)